core/src/main/java/io/questdb/Bootstrap.java

```diff
@@ -188,14 +190,20 @@ public CairoConfiguration getCairoConfiguration() {
             log.errorW().$(e).$();
             throw new BootstrapException(e);
         }
-
-        // metrics
         if (config.getMetricsConfiguration().isEnabled()) {
             metrics = Metrics.enabled();
         } else {
             metrics = Metrics.disabled();
             log.advisoryW().$("Metrics are disabled, health check endpoint will not consider unhandled errors").$();
         }
+
+        long writerMemoryLimit = config.getCairoConfiguration().getWriterMemoryLimit();
```

Comment:
writer memory limit is unused and can be removed, reporting it is confusing. May be @ideoma can comment?

Reply:
It seems to be used: https://github.com/questdb/questdb/blob/3729112805f7ee6ff878b57e366a119674ee765c/core/src/main/java/io/questdb/std/Unsafe.java#L320-L329

Reply:
oh, ok. I see. This is WAL-apply job thing. I don't see how is it practical to have two limits though? it is better to have one and have WAL Apply job obey common limit. What do you guys think?

Reply:
One limit is enough. Writer should obey the same limit, the only benefit it that it can estimate beforehand that it will exceed the limit and fail early.